### PR TITLE
Initial port of blackbox-exporter from kubernetes/charts repository [CO-209].

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

**Comment by philoserf** [comment]
Hard to follow w/o commentary. I would appreciate some commentary.

> No description provided.

then 

```
@davidewatson davidewatson added the DO_NOT_MERGE  label 13 days ago
 @davidewatson
Initial port of blackbox-exporter from kubernetes/charts repository.
39fdadb
 @davidewatson davidewatson removed the DO_NOT_MERGE  label 10 days ago
```

**Comment by davidewatson** [comment]
The title points you to https://samsung-cnct.atlassian.net/browse/CO-209 I believe this and the many links within are the more appropriate place for the information you are looking for.
